libs/sdk-backend-tiger/src/backend/workspace/styling/index.ts

```diff
@@ -3,17 +3,51 @@ import { IWorkspaceStylingService } from "@gooddata/sdk-backend-spi";
 import { IColorPaletteItem, ITheme } from "@gooddata/sdk-model";
 
 import { TigerAuthenticatedCallGuard } from "../../../types";
+import { getSettingsForCurrentUser } from "../settings";
 import { DefaultColorPalette } from "./mocks/colorPalette";
 import { DefaultTheme } from "./mocks/theme";
 
 export class TigerWorkspaceStyling implements IWorkspaceStylingService {
     constructor(private readonly authCall: TigerAuthenticatedCallGuard, public readonly workspace: string) {}
 
+    /**
+     * Checks if Theming needs to be loaded.
+     * Theming needs to be enabled by license entitlement
+     * by FF
+     * and activeTheme needs to be defined
+     *
+     * @returns boolean
+     */
+    private isThemable(activeThemeId: string | undefined, enablePantherTheming: boolean): boolean {
+        // TODO INE: add check for license entitlements
+        return !!activeThemeId && enablePantherTheming;
+    }
+
     public getColorPalette = async (): Promise<IColorPaletteItem[]> => {
         return this.authCall(async () => DefaultColorPalette);
     };
 
     public getTheme = async (): Promise<ITheme> => {
-        return this.authCall(async () => DefaultTheme);
+        const userSettings = await getSettingsForCurrentUser(this.authCall, this.workspace);
+        const activeThemeId = (userSettings.activeTheme as { id: string })?.id;
+
+        return this.isThemable(activeThemeId, !!userSettings.enablePantherTheming)
```

Comment:
Is `!!userSettings.enablePantherTheming` necessary? It's boolean anyway

Reply:
`ISettings.enablePantherTheming` is an optional field, so TypeScript thinks it can be undefined, even though it should not be undefined after FeatureHub resolver.

Reply:
Well, then maybe `userSettings.enablePantherTheming ?? false` would be nicer :)